UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformationAdmin.Web/appsettings.Development.json

```diff
@@ -7,8 +7,15 @@
     },
     "RadioNavigationalWarningsAdminContext": {
         "ConnectionString": ""
+
     },
     "RadioNavigationalWarningConfiguration": {
         "AdminListRecordPerPage": 20
+    },
+    "AzureAdB2C": {
+        "Instance": "",
+        "ClientId": "",
+        "Domain": ""
```

Comment:
Please remove AzureAdB2C section form admin app config as its not relevant to changes done for this PBI and also admin app requires AzureAd and not AzureAdB2C settings.

Reply:
removed AzureAdB2C section